platform-api/che-core-api-account/src/main/java/org/eclipse/che/api/account/server/dao/AccountDao.java

```
@@ -151,4 +154,50 @@ default boolean isWorkspaceRegistered(String workspaceId) throws ServerException
             return false;
         }
     }
+
+    /**
+     * Checks that account with given identifier exists.
+     *
+     * <p>This method covers the use-case where only account existence is important and
+     * account object itself is not needed, this is similar to {@link #getById(String)} + {@code try catch} just
+     * more convenient.
+     *
+     * <p>Example:
+     * <pre>
+     *     boolean exists = false;
+     *     try {
+     *         accountDao.getById(id);
+     *         exists = true;
+     *     } catch (NotFoundException nfEx) {
+     *         exists = false;
+     *     }
+     *
+     *     if (exists) {
+     *         // ..
+     *     }
+     *
+     *     // VS
+     *
+     *     if (accountDao.exists(id) {
+     *         // ..
+     *     }
+     * </pre>
+     *
+     * @param accountId
+     *         identifier of the account which check
+     * @return true if account exists or false if it does not
+     * @throws NullPointerException
+     *         when {@code accountId} is null
+     * @throws ServerException
+     *         when any other error occurs
+     * @see #getById(String)
+     */
+    default boolean exist(@NotNull String accountId) throws ServerException {
+        try {
+            getById(requireNonNull(accountId, "Required non-null account id"));
+            return true;
+        } catch (NotFoundException ignored) {
```

It is not ignored exception